src/db/asset_general.cc

```diff
@@ -615,7 +615,7 @@ db_reply_t
         return reply_delete2;
     }
 
-    auto reply_delete3 = delete_asset_links_all (conn, element_id);
+    auto reply_delete3 = delete_asset_links_to (conn, element_id);
```

Comment:
This will leave from-links in DB, please fix this.

Reply:
Based on the discussion in master branch PR #497 I agree this should work in the current state of code, and should be enforced with (lib)fty-common-db split.